src/BowlingScorer.js

```diff
@@ -0,0 +1,15 @@
+function BowlingScorer() {
+  this._score = 0;
+  this._first_frame = null;
+}
+
+BowlingScorer.prototype.score = function() {
+  return this._score;
+};
+
+BowlingScorer.prototype.addFrame = function(roll1, roll2) {
+  // have to check if first frame is null or not ...
+  // not sure about adding up frames?  some recursive call perhaps?
+  this._first_frame = new Frame(roll1, roll2)
```

Comment:
Identifier '_first_frame' is not in camel case.<br>Missing semicolon.<br>'Frame' is not defined.